superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -172,6 +172,33 @@ def _create_optimizer(self):
 
         return True
 
+    def _sync_result(self, result):
+        """Function to reduce the result to rank 0.
+
+        Args:
+            result (list): The result data to sync.
+
+        Return:
+            True if reduce result data successfully.
+        """
+        if not super()._sync_result(result):
+            return False
+
+        try:
+            if self._args.distributed_impl == DistributedImpl.DDP:
+                tensor = torch.as_tensor(result)
+                torch.distributed.reduce(tensor, 0, op=torch.distributed.ReduceOp.MAX)
+                result = tensor.tolist()
```

Comment:
It looks like that the time should be longer than before because we have the Reduce operation one more time.

Reply:
Yes, but it has no impact on our validation results.